script/crawl.sh

```diff
@@ -304,10 +304,6 @@ crawl "interlingue" \
 crawl "irish" \
   "https://github.com/kscanne/gaelspell" \
   "https://github.com/kscanne/gaelspell/releases/download/v5.1/hunspell-ga-5.1.zip"
-# Down, unfortunately :'(
-# crawl "italian" \
-#   "http://www.plio.it" \
-#   "https://iweb.dl.sourceforge.net/project/aoo-extensions/1204/14/dict-it.oxt"
 # Nothing:
```

Comment:
Wouldn’t getting it from `libreitalia.org` be better then?

I am intentionally using primary sources instead of pulling things in through an intermediary